package.json

```diff
@@ -38,6 +38,7 @@
     "gulp-nodemon": "^2.5.0",
     "gulp-rename": "^2.1.0",
     "gulp-sass": "^6.0.1",
+    "http-proxy-middleware": "^3.0.5",
```

Comment:
Feels like this should be in `devDependencies`, so that it can be skipped on Heroku and other hosts running prototypes in `NODE_ENV=production` mode?

Reply:
For now though, it's alongside `browsersync` and `gulp-nodemon`?